addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -45,7 +45,7 @@
         <field name="name">Work Orders</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">mrp.workorder</field>
-        <field name="view_mode">tree,form,gantt,calendar,pivot,graph</field>
+        <field name="view_mode">list,form,gantt,calendar,pivot,graph</field>
```

Comment:
what's the difference? 🤔
![Selection_531](https://user-images.githubusercontent.com/25005517/189673707-4fd25ba4-1b49-4a68-8840-4b4bb459dfaf.png)